src/App.test.js

```diff
@@ -3,6 +3,6 @@ import App from './App';
 
 test('renders learn react link', () => {
   render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
+  const linkElement = screen.getByText(/aries/i);
   expect(linkElement).toBeInTheDocument();
 });
```

Comment:
Excellent work getting this test to pass! React testing is my favorite library to test from because of its ease of use!